pubspec.yaml

```
@@ -1,18 +1,18 @@
 name: test_process
-version: 1.0.6-dev
+version: 2.0.0-nullsafety.0
```

@natebosch – might as well just go straight to stable, right?

---

Yeah we may as well.

```suggestion
version: 2.0.0
```

Still needs to be validated with google3, but we can have it prepped for a stable publish.